device-types/Fortinet/FG-100D.yaml

```diff
@@ -0,0 +1,66 @@
+---
+manufacturer: Fortinet
+model: FortiGate 100D
+slug: fg-100d
+part_number: FG-100D
+u_height: 1.0
+is_full_depth: false
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: DC Input for RPS
+    type: dc-terminal
+    maximum_draw: 63
+    allocated_draw: 53
+  - name: PS1
+    type: iec-60320-c14
+    maximum_draw: 63
+    allocated_draw: 53
+interfaces:
+  - name: dmz
+    type: 1000base-t
+  - name: ha1
+    type: 1000base-t
+  - name: ha2
+    type: 1000base-t
+  - name: mgmt
+    type: 1000base-t
+  - name: port1
+    type: 1000base-t
+  - name: port2
+    type: 1000base-t
+  - name: port3
+    type: 1000base-t
+  - name: port4
+    type: 1000base-t
+  - name: port5
+    type: 1000base-t
+  - name: port6
+    type: 1000base-t
+  - name: port7
+    type: 1000base-t
+  - name: port8
+    type: 1000base-t
+  - name: port9
+    type: 1000base-t
+  - name: port10
+    type: 1000base-t
+  - name: port11
+    type: 1000base-t
+  - name: port12
+    type: 1000base-t
+  - name: port13
+    type: 1000base-t
+  - name: port14
+    type: 1000base-t
+  - name: port15
+    type: 1000base-x-sfp
```

Comment:
Is this the default interface type? If so, then this is good to go and you can resolve this comment

Otherwise please keep in mind that all devices should be modeled with their default interface types out of the box, even if they can be changed.